docs/schema.md

```diff
@@ -1444,6 +1445,33 @@ Per-reporter staleness filter.
 </thead>
 <tbody>
 <tr>
+<td colspan="2" valign="top"><strong>AND</strong></td>
+<td valign="top">[<a href="#filterperreporterstaleness">FilterPerReporterStaleness</a>!]</td>
+<td>
+
+Apply logical conjunction on the given filtering criteria
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>OR</strong></td>
```

Comment:
This confused me a bit, because based on the other filters I would assume the fields here would be `reporter` and `stale_timestamp`.

Reply:
It was needed to add the logical conjunctions here because the custom staleness filters in graphql requires it. The PerReporterStalenessFilter did not had the logical conjunctions in the past